deploy/service_account.yaml

```diff
@@ -1,8 +0,0 @@
-apiVersion: v1
```

Comment:
please don't delete it.

Reply:
yes, I opened new PR to avoid the deletion. please ignore this PR, delete it now